google/cloud/bigquery/magics/magics.py

```diff
@@ -410,6 +437,15 @@ def _create_dataset_if_necessary(client, dataset_id):
         "Standard SQL if this argument is not used."
     ),
 )
+@magic_arguments.argument(
+    "--api_endpoint",
+    type=str,
+    default=None,
+    help=(
+        "The desired API endpoint, e.g., compute.googlepis.com. Defaults to this "
```

Comment:
Let's make the example bigquery-specific

```suggestion
        "The desired API endpoint, e.g., bigquery.googlepis.com. Defaults to this "
```